config/phenomenal_tools2container.yaml

```diff
@@ -72,10 +72,14 @@ assignment:
 - docker_image_override: isa-extractor
   docker_owner_override: phnmnl
   docker_repo_override: container-registry.phenomenal-h2020.eu
-  docker_tag_override: v1.1.0_cv1.0.2
+  docker_tag_override: dev_v1.1.2_cv1.0.3
```

Comment:
Hi, my curiosity: we use specific tags instead of :tatest during development ? 
I had thought we'd only fix versions when releasing, but I could be wrong.

Reply:
You're right, it's just me, I should have used latest.